ui.content/src/main/content/META-INF/vault/filter.xml

```diff
@@ -5,4 +5,5 @@
     <filter root="/content/dam/hclecomm" mode="merge"/>
     <filter root="/content/experience-fragments/hclecomm" mode="merge"/>
 	<filter root="/oak:index/hclecommLuceneIndex" mode="merge"/>
+    <filter root="content/cq:tags/ecommtags" mode="merge"/>
```

Comment:
@ReenaOmana: please add / before content else filter wont work

Reply:
Have made the changes